### DISCUSSION TITLE
Multiple nest missions select the same cave although there are other caves available

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15183] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![20241118182430](https://github.com/user-attachments/assets/1c36a8cf-a366-40de-a1b7-248f058c16b6)

![20241118182934](https://github.com/user-attachments/assets/85d4663d-c525-4b79-8f45-3885baeb87f6)

### Reproduction steps

1. select multiple nest missions
2. they likely select the same cave 

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
MORTAL KOMBAT !!!
